docs/components/modeler/desktop-modeler/telemetry/telemetry.md

```diff
@@ -139,50 +114,29 @@ Also in the case of BPMN diagrams, we add selected diagram metrics:
 }
 ```
 
-### Deployment event
-
-The `Deployment Event` is sent in following situations:
+### Deployment and start instance events
 
-- User deploys a BPMN diagram to Camunda Platform 7 or Camunda Platform 8
-- User deploys a DMN diagram to Camunda Platform 7
+The `Deployment Event` is sent in the following situations:
 
-The Deployment Event has the following core structure:
+- User deploys a BPMN or DMN diagram to Camunda Platform 7 or Camunda Platform 8
+- User deploys a Form to Camunda Platform 7
 
-```json
-{
-  "event": "deployment",
-  "diagramType": "[bpmn or dmn]",
-  "deployment": {
-    "outcome": "[success or failure]",
-    "context": "[deploymentTool or startInstanceTool]",
-    "executionPlatform": "[<target platform>]",
-    "executionPlatformVersion": "[<target platform version>]"
-  }
-}
-```
+The `Deployment Event` and `Start Instance` have the following properties:
 
-In case the diagram deployment was not successful, the error code returned from the Camunda Platform 7 will be added to the payload:
+- `diagramType`: BPMN, DMN, or Form
+- Engine profile:
+  - `executionPlatform`: <target platform\>
+  - `executionPlatformVersion`: <target platform version\>
 
-```json
-{
-  "deployment": {
-    "outcome": "failure",
-    "error": "DIAGRAM_PARSE_ERROR"
-  }
-}
-```
+We also send information regarding the outcome of the deployment. In case the diagram deployment was not successful, the `error` returned will be added to the payload.
```

Comment:
Instead of "We send information regarding..." do you mean "The event payload will also include..."? If not, essentially I'm just wondering **who** sends this information and to **where**?